File: Backend/src/ProEventosAPI/ProEventos.Domain/Models/Event.cs
using ProEventos.Domain.Identity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.Domain.Models
{
    public class Event
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? DateEvent { get; set; }
        public string Theme { get; set; }
        public int Capacity { get; set; }
        public string ImageURL { get; set; }
        public string CallNumber { get; set; }
        public string Email { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public IEnumerable<Part> Parts { get; set; }
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        public IEnumerable<SpeakerEvent> SpeakerEvents { get; set; }
    }
}

File: Frontend/ProEventos-App/src/app/models/Event.ts
import { Part } from './Part';
import { SocialMedia } from './SocialMedia';
import { Speaker } from './Speaker';

export interface Event {
 id: number;
 local: string;
 dateEvent?: Date;
 theme: string;
 capacity: number;
 imageURL: string;
 callNumber: string;
 email: string;
 parts: Part[];
 socialMedias: SocialMedia[];
 speakerEvents: Speaker[];
}
